### PR TITLE
CW luck check

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

File: tests/code_wars/test_luck_check.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    actual = luck_check("6838179")
+    expected = True
+
+    assert actual == expected, f"Returned {actual} instead of {expected}"
+
+
+def test_luck_check_empty_string():
+    with pytest.raises(ValueError) as exc_info:
+        luck_check("")
+        assert exc_info.value.args[0] == "empty string"
```
Comment (W-Ely) [inline tests/code_wars/test_luck_check.py:31]: The `assert` needs to be pulled outside the `with` block.  To see what I mean you can change `== "empty string"` too `== "foo"`, it should fail but won't.

As soon as luck_check is ran the value error is thrown. Pytest catches this error and exits the with context block.

**Review by W-Ely — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -87,7 +87,94 @@
   * Tests: `tests/code_wars/test_count_smileys.py`
   * URL: [Challenge url](https://www.codewars.com/kata/583203e6eb35d7980400002a/python)
   * Run Tumes: O(n)
+<<<<<<< HEAD
```
Comment (W-Ely) [inline README.md]: looks like this conflict wasn't resolved completely.

File: README.md [hunk 3]
```diff
@@ -87,7 +87,94 @@
...
 ### Luck Check (5kyu)
   * Module: `code_challenges/code_wars/luck_check.py`
   * Tests: `tests/code_wars/test_luck_check.py`
-  * URL: [Challenge url](https://www.codewars.com/kata/5314b3c6bb244a48ab00076c/python)
\ No newline at end of file
+  * URL: [Challenge url](https://www.codewars.com/kata/5314b3c6bb244a48ab00076c/python)
+=======
```
Comment (W-Ely) [inline README.md]: .

File: README.md [hunk 4]
```diff
@@ -87,7 +87,94 @@
...
+  * Module: `code_challenges/advent_of_code/tuning_trouble.py`
+  * Tests: `tests/advent_of_code/test_tuning_trouble.py`
+  * URL: [challenge url](https://adventofcode.com/2022/day/6)
+## Hacker Rank
+### Find String:
+  * Module: `code_challenges/hacker_rank/find_string.py`
+  * Tests: `tests/hacker_rank/test_find_string.py`
+  * URL: [challenge url](https://www.hackerrank.com/challenges/find-a-string/problem?isFullScreen=true)
+
+>>>>>>> main
```
Comment (W-Ely) [inline README.md]: .

**Comment by aepaysinger** [comment]
when i go to that branch and open the README it doesnt look like that

**Review by W-Ely — APPROVED** [review]

**Pull request merged**